mk/re.mk

```diff
@@ -725,6 +726,11 @@ rpm:    tar
 # - system installation
 #
 
+ifneq ($(SYSROOT),/usr)
+LIBRE_PATH := $(shell [ -f $(SYSROOT)/include/re/re.h ] && \
+	echo "$(SYSROOT)")
+endif
+
```

Comment:
I think we should guarantee the prioritised order for relative path re search,
Like this (untested):

```mk
ifeq ($(LIBRE_PATH),)
LIBRE_PATH := $(shell [ -d ../re ] && echo "../re")
endif

ifeq ($(LIBRE_PATH),)
LIBRE_PATH := $(shell [ -f $(SYSROOT)/include/re/re.h ] && \
	echo "$(SYSROOT)")
endif
```

Reply:
So the new order we should document:

# - relative path (../re)
# - custom sysroot
# - local installation (/usr/local)
# - system installation (/usr)